### ISSUE TITLE
No points filter option

### ISSUE THREAD

**Issue by DistantQ** [CnCNet/xna-cncnet-client#340]
**Create an option that allows players to QM without points filter**

-match closest in range but no restriction on points spread
-max points won/lost = 80pts (I think this is the maximum currently?)
-both players must enable this option to be eligible

This would reduce/eliminate wait times in certain cases:
More available matches.
Would allow two players to instantly match if no other qm matches were available (e.g. Marko with 2000pts vs Gino with 1400pts - I have seen this kind of scenario very often as neither player is willing to wait out the widening pts filter.)
Allow players to potentially rank up faster who may have gotten a late start at the end of the month.

With only 1 nickname per account I dont think this can easily be abused. If someone is trying to target a high ranked player, the higher ranked player can disable the feature whenever they choose.

**Comment by Metadorius** [comment]
I don't think this is related to XNA CnCNet client, at least yet. One may reopen if this will change.

**Comment by alexp8** [comment]
I am reopening this
